src/plugins/console/public/application/hooks/use_send_current_request_to_es/send_request_to_es.ts

```diff
@@ -65,23 +67,29 @@ export function sendRequestToES(args: EsRequestArgs): Promise<ESRequestResult[]>
       } // append a new line for bulk requests.
 
       const startTime = Date.now();
-      es.send(esMethod, esPath, esData).always(
-        (dataOrjqXHR, textStatus: string, jqXhrORerrorThrown) => {
-          if (reqId !== CURRENT_REQ_ID) {
-            return;
-          }
 
-          const xhr = dataOrjqXHR.promise ? dataOrjqXHR : jqXhrORerrorThrown;
+      try {
```

Comment:
I'm most concerned about this file being changed without any test coverage. There is some logic about processing successful and failed requests that we should be testing I think.